tensorbay/client/diff.py

```diff
@@ -198,12 +199,12 @@ class DatasetDiff(Sequence[SegmentDiff], NameMixin):  # pylint: disable=too-many
 
     """
 
-    _repr_attrs = ("name", "action")
+    _repr_attrs = ("action",)
 
-    def __init__(self, name: str) -> None:
+    def __init__(self, name: str, segments: PagingList[SegmentDiff]) -> None:
         super().__init__(name)
 
-        self._segments: SortedNameList[SegmentDiff] = SortedNameList()
+        self._segments: SortedNameList[SegmentDiff] = segments  # type: ignore[assignment]
```

Comment:
Why the `self._segments` is of `SortedNameList[SegmentDiff]`?

Reply:
Cuz supporting getting item by name